### PR TITLE
Make identification e2e tests pass

### LINKED ISSUE #1212: Get Identification section spec tests passing
Reference: https://docs.google.com/document/d/1HoXI8i4wBcjWq_elD39BuNYdesi8cjHnnERLk4JcfT4/edit#bookmark=id.yw3fljo9pvsh

For this task, it is not just sufficient to get the existing Identification section tests passing, but to validate that those test cover the section, and augment them as necessary.

### PR DESCRIPTION
**Pull request by gidjin**
Fixes #1212 by adding a passing spec. Also this generated #1274 to cover failure example at least minimally

### CODE REVIEW

**Comment by gidjin** [comment]
I've been running the test on it's own and it passes as expected. It failed when run with the entire suite I didn't have time this round to look at why, but once all the tests are passing individually we should circle back to ensure the whole suite passes

**Comment by gidjin** [comment]
Yes I've been thinking of pulling the code out into page objects so that any test can reuse the code to fill in the form. I wasn't sure on the best approach for that with Nightwatch, but there is support for page objects. I'm open to other suggestions on making them more reusable, I would like to see a test that went through one of the NBIB test scenarios too, but probably we want some basic page objects and fixtures before doing that.

File: README.md [hunk 1]
```diff
@@ -68,6 +68,8 @@ Setup steps:
 1. Disable *SAML\_ENABLED* auth and enable *BASIC\_ENABLED*
 1. Edit /etc/hosts file __OPTIONAL__
   * Add _web_ and _api_ to the end of the localhost line in /etc/hosts so you don't have to keep changing the values of API\_REDIRECT or API\_BASE\_URL in .env file back to localhost
+1. Run __make run__ to ensure services are restarted
+1. Run the purge directions from docs/test-scenarios.md to clear out old data
```
Comment (ryanhofdotgov) [inline README.md]: @gidjin Markdown doesn't seem to be numbering the steps as you would expect:

screen shot 2018-12-17 at 1 56 44 pm

Comment (gidjin) [reply README.md]: Apparently not. I've updated it, I couldn't get sublists to number properly so I took them out since it always started over.

**Pull request merged**